internal/server/templates/index.html

```diff
@@ -4,7 +4,39 @@
 	<meta charset="utf-8" />
 	<title>scrape</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
 	<style>
+		body, html {
+			margin: 0;
+			padding: 0;
+			height: 100vh;
+		}
+		.page-container {
+			display: flex;
+			flex-direction: column;
+			gap: 0.6rem;
+			box-sizing: border-box;
+			height: 100%;
+			padding: 0.6rem;
+		}
+		.controls-container {
+			background-color: #fff;
+			box-sizing: border-box;
+		}
+		.response-container {
+			width: 100%;
+			min-width: 768px;
+			flex: 1;
+			overflow: auto;
+			border: 0.1rem inset #ccc;
+			padding-block: 0.2rem;
+			padding-inline: 0.4rem;
+			box-sizing: border-box;
+		}
+		pre {
+			white-space: pre-wrap;
+			width: 100%;
+		}
 		label {
 			display: block;
 			font: 1rem 'Verdana'
```

Comment:
(nit) Suggestion: Add [multiple font families](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#some_common_font_families) for better font choice fallback. 

```
font-family: Verdana, Arial, Helvetica, sans-serif;
```

Reply:
Updated in 7362a59092454278210ad77de08821a46ca3fc54